### PR TITLE
fix(addon-doc): fix import package

### LINKED ISSUE #239: Addon-doc, can't not import TUI_DOC_PAGE_MODULES
# 🐞 Bug report

### Reproduction

I am trying to use Addon-doc. I read [docs](https://github.com/TinkoffCreditSystems/taiga-ui/tree/main/projects/addon-doc#how-to-install). 
When i try to execute ng serve, the error is occurred.
Screenshot 2021-02-24 at 19 08 24

### Versions

-   Angular 11.0.5
-   Package [e.g. @@taiga-ui/addon-doc@2.5.0]

### PR DESCRIPTION
**Pull request by nikitadubinsk**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [x] Bugfix
-   [ ] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

## What is the current behavior?

Closes #239 

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [ ] No

### CODE REVIEW

File: projects/addon-doc/README.md [hunk 1]
```diff
@@ -101,12 +101,12 @@ npm i @taiga-ui/addon-doc
     ```typescript
     import {NgModule} from '@angular/core';
     import {RouterModule} from '@angular/router';
-    import {generateRoutes, TUI_DOC_PAGE_MODULES} from '@taiga-ui/addon-doc';
```
Comment (MarsiBarsi) [inline projects/addon-doc/README.md:104]: let's also deprecate TUI_DOC_PAGE_MODULES constant and replace it in all samples in demo

**Review by MarsiBarsi — CHANGES_REQUESTED** [review]

File: projects/addon-doc/README.md (hunk 1 above)
Comment (nikitadubinsk) [reply projects/addon-doc/README.md:104]: I have to change absolutely in all places (there it is in very many components, etc.)?

**Review by MarsiBarsi — APPROVED** [review]
I am ready to approve it but @waterplea please take a look: is it okay to remove the old constant without mentioning a breaking change? I think a very few people used it

**Comment by waterplea** [comment]
> I am ready to approve it but @waterplea please take a look: is it okay to remove the old constant without mentioning a breaking change? I think a very few people used it

I think it's fine since it doesn't work :)

**Pull request merged**